### PR TITLE
conmon: do not create oom file under cwd

### PR DESCRIPTION
**Pull request by giuseppe**
instead use the bundle path to create the second (shurgh!) file since this is what CRI-O uses.

Closes: https://github.com/containers/conmon/issues/504

### CODE REVIEW

File: src/cgroup.c [hunk 1]
```diff
@@ -306,33 +306,39 @@ gboolean check_cgroup2_oom()
...
- * this can be used for v1 and v2 OOMS
+static int create_oom_file(const char *base_path, const char *path_type)
+{
+	if (base_path == NULL || base_path[0] == '\0')
+		return 0;
+
+	_cleanup_free_ char *ctr_oom_file_path = g_build_filename(base_path, "oom", NULL);
+	_cleanup_close_ int ctr_oom_fd = open(ctr_oom_file_path, O_CREAT | O_CLOEXEC, 0666);
+	if (ctr_oom_fd < 0) {
+		nwarnf("Failed to write oom file to the %s path", path_type);
```
Comment (Luap99) [inline src/cgroup.c:325]: If I read such a message using `ctr_oom_file_path` would make the most sense as a user is likely not knowing where this path would be and if there is a full file path a user could check this directory quickly.

**Review by Luap99 — COMMENTED** [review]
one nit, otherwise LGTM

File: src/cgroup.c (hunk 1 above)
Comment (giuseppe) [reply src/cgroup.c]: replaced `path_type` with `base_path` itself

**Review by Luap99 — COMMENTED** [review]
LGTM

File: src/cgroup.c [hunk 2]
```diff
@@ -306,33 +306,39 @@ gboolean check_cgroup2_oom()
...
 				last_counter = counter;
 		}
 		return G_SOURCE_CONTINUE;
 	}
 	return G_SOURCE_REMOVE;
 }
 
-/* write the appropriate files to tell the caller there was an oom event
- * this can be used for v1 and v2 OOMS
+static int create_oom_file(const char *base_path)
```
Comment (haircommander) [inline src/cgroup.c:317]: nit: can this be defined underneath where it's called?

Comment (giuseppe) [reply src/cgroup.c:331]: done

**Comment by haircommander** [comment]
let's try to get ci running to double check this works with crio https://github.com/containers/conmon/pull/515

**Comment by haircommander** [comment]
@giuseppe can you rebase to get test fixes please?

**Comment by giuseppe** [comment]
rebased

**Comment by haircommander** [comment]
thanks!

**Pull request merged**